### ISSUE TITLE
Pages missing meta attributes

### ISSUE THREAD

**Issue by clacote** [mixitconf/mixit#229]
Pages without title:
- https://mixitconf.org/mixteen
- https://mixitconf.org/come

Duplicate titles (seen in https://www.google.com/webmasters/tools/html-suggestions?hl=en&siteUrl=https://mixitconf.org/&probtype=9&pl=eyJ0YXJnZXRJZCI6Imh0bWwtc3VnZ2VzdGlvbnMiLCJwYXJhbXMiOnsic2l0ZVVybCI6Imh0dHBzOi8vbWl4aXRjb25mLm9yZy8iLCJobCI6ImVuIn19)
- /2017/design
- /2017/hacktivism
- /2017/learn
- /2017/tech
- /2017

Also, `/en` seems no recognised as language path by Google: _duplicate titles_ for every canonical and its `/en`alternative .

**Comment by sdeleuze** [comment]
I added missing titles via https://github.com/mixitconf/mixit/commit/73da88ce019b7426656d8cbbd5ba6ccd672c319a.

I don't understand why `/2017/foo` pages are reported with duplicated titles, try it yourself you will see they are differents.

I also I don't understand why Google does not recognize `en` content correctly since we have followed guidelines (html lang attributes + `<link rel="alternate" hreflang=""> tags).

Any thoughts?
